news_items/011_python_autograder/index.html

```diff
@@ -0,0 +1,68 @@
+
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+ .code-block {
+     font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+     background: #f0f0f0;
+     width: 70%;
+     margin-left: auto;
+     margin-right: auto;
+ }
+ .code-block > pre {
+     padding: 0.5rem;
+     display: block;
+     margin-bottom: 0px;
+     line-height: 125%;
+     overflow: wrap;
+     overflow-wrap: normal;
+ }
+</style>
+
+<p>
+    Thanks to <a href="https://github.com/nicknytko">Nicolas Nytko</a> (CS MS '22), <a href="https://github.com/bowmnath">Nathan Bowman</a>, and <a href="https://github.com/inducer">Andreas Kl&ouml;ckner</a>, vast improvements have been made to the default Python autograder.  Among some of the added features include the ability to autograde various datatypes like NumPy arrays, Pandas DataFrames, and Matplotlib plots:
+</p>
+
+<img src="plot_grading.png" width="756px">
```

Comment:
I hate to ask this, but could we get a more interesting image? 😄 Even just a sine wave? Or `sin(1/x)`? Anything except a straight line! If it could be made less tall then that would also be great (that is, just as wide, but not as tall).

Reply:
Hah, yeah that's a pretty boring plot looking back at it now... I'll get something nicer to look at.